setup.py

```diff
@@ -143,9 +144,9 @@
         'Programming Language :: Python :: 2',
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.3',
-        'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: Implementation :: CPython',
+        'Programming Language :: Python :: Implementation :: PyPy',
```

Comment:
PyPy is failing in CI, so I think we shouldn't declare PyPy-readiness... Better to delete it for now.

```
RuntimeError: cryptography 1.0 is not compatible with PyPy < 2.6. Please upgrade PyPy to use this library.
```